docs/content/en/docs/reference/artifacts.md

```diff
@@ -139,6 +141,7 @@ To use `image-builder` you must meet the following prerequisites:
   * Bare Metal only: Run on a bare metal machine with virtualization enabled
```

Comment:
We need to add this for Cloudstack too. We need baremetal machine with virtualization enabled to build cloudstack images too.

Reply:
Done.